src/utils.js

```diff
@@ -90,5 +97,6 @@ function parseSolutionSeedPhrase(data, gridData) {
 module.exports = {
   mungeBlockchainCrossword,
   viewMethodOnContract,
-  parseSolutionSeedPhrase
+  parseSolutionSeedPhrase,
+  b64ToUtf8: b64toUtf8
```

Comment:
Not too familiar with JS, but this looks weird. I'm not sure if this is fully required, since wouldn't `b64toUtf8` suffice instead of `b64toUtf8: b64toUtf8` for this export?